test/tests/redfish10/test_redfish10_api_taskservice.py

```diff
@@ -136,8 +132,7 @@ def test_redfish_v1_taskservice_tasklist(self):
         for member in members:
             tasklist.append(member['Id'])
         if fit_common.VERBOSITY >= 2:
-           print ("Task Service contains {0} tasks.".format(len(tasklist)))
-
+            print ("Task Service contains {0} tasks.".format(len(tasklist)))
```

Comment:
I want to convert this to flogger, but not in this PR. I think that is too much to review and this is time-critical. There is a lot if this stuff all through the legacy code base. Let's make this its own story and chalk it up to technical debt.